packages/commercetools/api-client/src/types/setup.ts

```diff
@@ -56,3 +56,37 @@ export interface CustomerCredentials {
   username: string;
   password: string;
 }
+
+// --
+
+export interface Config {
+  api: ApiConfig;
+  currentToken?: Token;
+  customOptions?: ApolloClientOptions<any>;
+  currency: string;
+  locale: string;
```

Comment:
we agreed on different interface in the task and slack - https://github.com/DivanteLtd/vue-storefront/issues/4777